content/en/docs/developer/run-ut.md

```diff
@@ -0,0 +1,61 @@
+---
+draft: false
+linktitle: Unit Test Guide
+menu:
+  docs:
+    parent: developer guide
+    weight: 10
+title: Run Unit test
+toc: true
+type: docs
+---
+
+Compiling Kmesh directly in the operating system requires a certain [kernel version](https://github.com/kmesh-net/kmesh/blob/main/docs/kmesh_support.md) of the operating system. Therefore, in order to allow all operating systems to run Kmesh's ut, Kmesh provides two ways to do so. One to run the go unit in docker and one to run the go unit locally.
+
+Developers of unsupported kernel version can run go unit test in docker through script. Developers of supported version can run go unit test locally through script.
```

Comment:
All docker containers share the host kernel.

Why running in the docker container could solve the problem of running UT in the unsupported kernel version?

Reply:
I think it might be on lower os version, the include files doesnot match, while the build docker contains that